.github/workflows/publish.yml

```diff
@@ -8,13 +8,58 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    # TODO: version in and env var
     steps:
       - uses: actions/checkout@v2-beta
-      - name: Build the Docker image
-        run: docker build .
-      - name: Publish to Registry
+      - name: Build and publish to Docker Hub
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
           name: smartcontract/builder:1.0.35
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      # Push to ECR
+      - name: install aws cli
```

Comment:
Let's DRY this out via https://github.com/smartcontractkit/chainlink/pull/3212#pullrequestreview-460355030

Reply:
Agreed. I'm working on it! Thanks for the review!